djhtml/djhtml.py

```diff
@@ -0,0 +1,12 @@
+from .modes import HTML
```

Comment:
It's confusing to introduce another module called `djhtml.djhtml`. I'd prefer to place methods like this in `__init__.py` instead. However, see my comment below.